### DISCUSSION TITLE
Custom thumb for Switch component

### DISCUSSION THREAD

**Discussion by ajhurliman** [GeekyAnts/NativeBase#5483] [Feature request]
### Description

I'd like the ability to specify a custom thumb for the Switch component. Specifically, when the switch's value is isChecked={true} I'd like a checkmark icon to appear over the thumb. Would this be very difficult?

<img width="140" alt="Screen Shot 2022-10-04 at 1 48 01 PM" src="https://user-images.githubusercontent.com/8345338/193924761-9dc6c3c0-afb1-48c9-885a-35643b7f2b61.png">

### Problem Statement

To allow users to further customize the Switch component

### Proposed Solution or API

Could add a prop `customThumb={<div>...</div>}`

**Comment by ankit-tailor** [comment]
Hey @ajhurliman, Thanks for your suggestions. We'll consider adding it in future releases.
